### DISCUSSION TITLE
How to update to a specific version?

### DISCUSSION THREAD

**Discussion by Rolzzz** [valinet/ExplorerPatcher#3902] [General]
Hello there, I've got a new laptop and I want to update to version 22621.3880.66.5 as that seemed to be the last vesion that didn't send all my AV in a tizz

Is there a way I can transfer specific files over from my old laptop to my new one?

Or can I find the specific version here and just install that one?

Sorry I'm not super well versed with using Github.

thanks for any help you can provide :)

**Comment by Amrsatrio** [comment]
https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3880.66.5_5094108

You can turn off updates in EP Properties > Updates.

But beware, that version will have non working 10 Start menu right click on latest 22H2, 23H2, and 24H2 updates. Also beware, one of the files it drops (ep_weather_host_stub.dll) has been flagged by MS.

If you cannot go with the process of adding EP's directories into exclusions, I recommend that you stop using ExplorerPatcher on your new laptop and look for alternatives instead.

**Reply by Rolzzz** [reply]
thank you!!
